processor/src/test/java/org/mapstruct/ap/test/nestedbeans/mixed/_target/WaterQualityOrganisationDto.java

```diff
@@ -16,33 +16,16 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.test.nestedtargetproperties.source;
+package org.mapstruct.ap.test.nestedbeans.mixed._target;
 
 /**
  *
  * @author Sjaak Derksen
  */
-public class FishTank {
+public class WaterQualityOrganisationDto {
 
-    private Fish fish;
-    private WaterPlant plant;
     private String name;
-
-    public Fish getFish() {
-        return fish;
-    }
-
-    public void setFish(Fish fish) {
-        this.fish = fish;
-    }
-
-    public WaterPlant getPlant() {
-        return plant;
-    }
-
-    public void setPlant(WaterPlant plant) {
-        this.plant = plant;
-    }
+    private String approval;
```

Comment:
Indirectly by not generating an error for it because it is not mapped. However, I think that you are right, we need to make sure that it is actually `null` in the tests as well.